### PR TITLE
Foam armblade suicide

### LINKED ISSUE #20390: Foam Armblade Suicide Request
#### Description of issue
Foam Armblade does the unique absorb message but has no attached suicide, it'd be interesting and amusing if the Foam Armblade caused you to noisily DNA absorb yourself upon suicide, effectively husking you.

### PR DESCRIPTION
**Pull request by CptWad**
closes #20390
I think I did this right

:cl:
 * rscadd: Committing suicide with a foam armblade is more interesting now

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -396,6 +396,10 @@
 	item_state = "armblade"
 	attack_verb = list("pricked", "absorbed", "gored", "stung")
 	w_class = W_CLASS_MEDIUM
+	/obj/item/toy/foamblade/suicide_act(mob/user)
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/game/objects/items/toys.dm]: What is this tabulation

Comment (Exxion) [reply code/game/objects/items/toys.dm]: In addition to that the lack of a line break makes me angry

Comment (CptWad) [reply code/game/objects/items/toys.dm]: did I do it wrong

**Comment by Pathid** [comment]
>rscadd: Committing suicide with an armblade is more interesting now

This really isn't accurate. Especially as it's possible to get a real armblade from Summon Swords and have it stuck to your hand.

**Comment by CptWad** [comment]
oh right

**Comment by Durgen** [comment]
lmao

**Pull request merged**